### PR TITLE
Add tests for Chaos Portal

### PR DESCRIPTION
**Pull request by jtrussell**
Tests added to cover the Chaos Portal + Duskwitch (omega cards) interaction mentioned in #690.

It appears that the underlying issue was actually fixed by b393d6c and just hasn't made it to production yet.

### CODE REVIEW

**Comment by jtrussell** [comment]
I wasn't sure if referencing non-cota cards in `tests/../01-Core` was how folks would like to handle this or if we'd rather see this in `tests/.../02-AoA`.

**Comment by cryogen** [comment]
> I wasn't sure if referencing non-cota cards in `tests/../01-Core` was how folks would like to handle this or if we'd rather see this in `tests/.../02-AoA`.

As long as the test is in the folder for the set where the card first appeared, that's fine.  Don't mind which cards various tests refer to.  One trap to avoid falling into is testing unrelated things in your tests.  Eg if you're testing a creature's fight ability fires, you don't want to test that bad penny goes back to hand after she dies, unless that's important to the interaction at hand.  Some of our tests do fall into this trap

File: test/server/cards/01-Core/ChaosPortal.spec.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                        inPlay: ['chaos-portal'],
+                        discard: ['duskwitch', 'bad-penny']
+                    }
+                });
+            });
+
+            it('should prompt to choose a house', function() {
+                this.player1.useAction(this.chaosPortal);
+                expect(this.player1).toHavePrompt('Chaos Portal');
+                expect(this.player1.currentButtons).toContain('brobnar');
```
Comment (cryogen) [inline test/server/cards/01-Core/ChaosPortal.spec.js]: expect(this.player1).toHavePromptButton('brobnar') is a helper method that exists that would be appropriate here

Comment (jtrussell) [reply test/server/cards/01-Core/ChaosPortal.spec.js]: Sweet - I'll change that up 👍. I have to admit I pretty much copied this straight from `Arise!` and didn't really look at it 😅.

Comment (cryogen) [reply test/server/cards/01-Core/ChaosPortal.spec.js]: The framework has evolved over time, so it's entirely possible that this is a new thing, so don't worry :)

**Comment by jtrussell** [comment]
Looks like the TC build failed. Locally, all (not skipped) tests pass, linting checks out, and the webpack build runs without error.

I don't suppose there's any way I could get at those error messages on TC?

**Comment by cryogen** [comment]
You can see if you click details and log in as a guest. However in this instance the issue is because my deployment server is offline, so nothing for you to do :) it will fix itself shortly!

**Pull request merged**